src/clients/customer/lib/ui/dumb_widgets/autocomplete_listItem.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:box_ui/box_ui.dart';
```

Comment:
Widgets should all be built in the `box_ui` package. We know that when using the driver app or the Restaurant management app the user will have the same auto complete UI to select their address or restaurants address.

Reply:
All right, Dane will update the changes